GNUmakefile

```diff
@@ -345,6 +345,11 @@ llvm:
 	-$(MAKE) -j$(nproc) -f GNUmakefile.llvm
 	@test -e afl-cc || { echo "[-] Compiling afl-cc failed. You seem not to have a working compiler." ; exit 1; }
 
+llvm-build-test:
```

Comment:
added new target. because old `make llvm` will ignore error (thus ci won't fail for broken code)